app/config.js

```diff
@@ -2,8 +2,7 @@
 
 module.exports = {
   // Service name
-  serviceName: 'NHS.UK prototype kit documentation',
-
+ serviceName: 'NHS.UK prototype kit documentation',
```

Comment:
Restore indenting here.

```suggestion
  serviceName: 'NHS.UK prototype kit documentation',
```